esp32/modules/services.py

```diff
@@ -1,59 +1,250 @@
-import uos
-
 # File: services.py
+# Version: 2
+# API version: 1
 # Description: Background services for SHA2017 badge
 # License: MIT
 # Authors: Renze Nicolai <renze@rnplus.nl>
 #          Thomas Roos   <?>
 
-def setup():
+import uos, ujson, easywifi, easyrtc, time, appglue, deepsleep, ugfx, badge, machine
+
+services = [] #List containing all the service objects
+loopCallbacks = {} #Dict containing: {<FUNCTION>:<Wifi required on next run>}
+drawCallbacks = [] #List containing draw functions
+
+def setup(pmCb=False, drawCb=False):
```

Comment:
`None` makes more sense as a default here, as the type of the parameter is a function, now it looks like a boolean